sagemaker-clarify/clarify-explainability-inference-pipelines/README.md

```diff
@@ -0,0 +1,25 @@
+## Credit risk prediction and explainability with Amazon SageMaker
+
+This example shows how to user SageMaker Clarify to run explainability jobs on a SageMaker hosted inference pipelines. 
+
+Below is the architecture diagram used in the solution:
+
+![alt text](clarify_inf_pipeline_arch.png)
+
+
+The notebook performs the following steps:
+
+1. Prepare raw training and test data
+2. Create a SageMakerProcessing job which performs preprocessing on the raw training data and also produce an SKlearn model which is reused while deployment.
```

Comment:
Grammar revisions - `Create a SageMaker Processing job which performs preprocessing on the raw training data and also produces an SKlearn model which is reused for deployment.`

Reply:
fixed